.emm-local.yml

```diff
@@ -0,0 +1 @@
+evg_project: evg-module-manager
```

Comment:
Do we want to include this in the repo? Or let everyone generate their own?

Reply:
I was thinking about that. Since everyone that uses this repo would want to use the same value, it kind of makes sense to check in. But I also supported the modules directory in the same way and that would be different by user. So, maybe we should remove it.